0.16.0-release-notes.md

```diff
@@ -31,3 +31,10 @@ New features in the interactive editor:
 
 -   The editor now uses a DSL for filtering items in completion, history
     listing, location and navigation modes.
+   
+-   A new `file:` module to work with files.
+
+# Deprecated features
+
+-   The `builtin:fopen` and `builtin:fclose` commands are deprecated. Use 
+    the equivalent commands in the new `file:` module.  
```

Comment:
Please reformat this file with prettier, as documented in CONTRIBUTING.md. The CI error message also contains a diff, you can apply it manually if you find installing prettier too cumbersome.

Reply:
Yes sir, I will take care of this.

Reply:
Didn't mean to push this one yet, I will get back to it soon. Need to format it.